### ISSUE TITLE
problem in test result

### ISSUE THREAD

**Issue by LegendBC** [mohhao/TF-Keras-ThunderNet#2]
Thanks for your great work! And I haved trained your model on pascal2007, but the result in test shows no class only background. I haven't changed anything else except for the function get_data(input_path). Can you give me some advice? thanks!

**Comment by LegendBC** [comment]
It's of great help that you give a more specific readme about your environment configuration.

**Comment by wusonglin-ali** [comment]
> It's of great help that you give a more specific readme about your environment configuration.

你好，你现在解决这个问题了吗

**Comment by hsl8856** [comment]
> Thanks for your great work! And I haved trained your model on pascal2007, but the result in test shows no class only background. I haven't changed anything else except for the function get_data(input_path). Can you give me some advice? thanks!

The test results are all background questions.,. Have you solved it? 
What is the final convergence result of your training loss function(RPN,CLass)? thanks

**Comment by kris1025** [comment]
I also encounter this problem .Does anyone know how to solve this problem?

**Comment by cfeeney5** [comment]
There is a bug in the PSRoiAlignPooling function in detector.py in the latest commit. It will work if the RoiPoolingConv function is used instead.

**Comment by Threeki1019** [comment]
> There is a bug in the PSRoiAlignPooling function in detector.py in the latest commit. It will work if the RoiPoolingConv function is used instead.

hi! can you tell me how to use RoiPoolingConv function

**Comment by cfeeney5** [comment]
@Threeki1019 I didn't fix it I just used RoiPoolingConv function instead

**Comment by Threeki1019** [comment]
yeah but i test.py also only bg




------------------ Original ------------------

**Comment by Threeki1019** [comment]
if use&nbsp;RoiPoolingConv function ，you get the correct results ？




------------------ Original ------------------

**Comment by cfeeney5** [comment]
Hi,
I changed over ROIPoolingConv and I could train the network but the results were only OK. The code suggests initializes the weights from VGG-16 but I dont think it is correct because the layer names do not match. How many images are in your training set? 
The bounding box threshold in this commit is 0 (see https://github.com/mohhao/TF-Keras-ThunderNet/blob/d85ccf9f44b9470ff8459ba8c47f2ec6cc6a3e4a/test.py#L130 ) which should be 0.5 or so.

**Comment by Threeki1019** [comment]
I used your advice, but my training result loss has been decreasing, but the ACC remains the same. Train on voc2007




------------------ Original ------------------

**Comment by Threeki1019** [comment]
i used the vgg16weight，but i think it didnt work。




------------------ Original ------------------
